selinux/ipa.te

```diff
@@ -147,13 +147,17 @@ auth_use_nsswitch(ipa_helper_t)
 
 files_list_tmp(ipa_helper_t)
 
+init_read_state(ipa_helper_t)
+init_stream_connect(ipa_helper_t)
+
 ipa_manage_pid_files(ipa_helper_t)
 ipa_read_lib(ipa_helper_t)
 
 logging_send_syslog_msg(ipa_helper_t)
 
 optional_policy(`
     dirsrv_stream_connect(ipa_helper_t)
+    dirsrv_systemctl(ipa_helper_t)
```

Comment:
``dirsrv_systemctl`` is a new interface and not yet availabe. If I understand SELinux correctly, you have to put it in a separate ``optional_policy`` block. Otherwise ``dirsrv_stream_connect(ipa_helper_t)`` will not take affect on systems without ``dirsrv_systemctl`` interface.

Reply:
You are right, so changing as I am not fully aware of the workflow or relation selinux-policy.